db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBPersistentSubscriptionsClient.java

```diff
@@ -8,204 +8,375 @@
 import java.util.concurrent.ExecutionException;
 import java.util.function.Function;
 
+/**
+ * Represents a client to a single node. A client instance maintains a full duplex communication to EventStoreDB.
+ * Many threads can use an EventStoreDB client at the same time or a single thread can make many asynchronous requests.
+ */
 public class EventStoreDBPersistentSubscriptionsClient extends EventStoreDBClientBase {
-    private static final ObjectMapper mapper = new ObjectMapper();
 
     private EventStoreDBPersistentSubscriptionsClient(EventStoreDBClientSettings settings) {
         super(settings);
     }
 
-    public static EventStoreDBPersistentSubscriptionsClient createToStream(EventStoreDBClientSettings settings) {
+    /**
+     * Creates a persistent subscription client instance.
+     */
+    public static EventStoreDBPersistentSubscriptionsClient create(EventStoreDBClientSettings settings) {
         return new EventStoreDBPersistentSubscriptionsClient(settings);
     }
 
+    /**
+     * Creates a persistent subscription group on a stream.
+     *
+     * <p>
+     * Persistent subscriptions are special kind of subscription where the server remembers the state of the

```

Comment:
**SUGGESTION:** As it's a common misconception, it may be worth add here, that they don't guarantee ordering.

Reply:
Applied both suggestions.